lab_3_ann_retriever/main.py

```diff
@@ -334,6 +445,13 @@ def _index_document(self, document: str) -> Vector | None:
 
         In case of corrupt input arguments, None is returned.
         """
+        if not isinstance(document, str):
+            return None
+        document_tokenized = self._tokenizer.tokenize(document)
+        if not isinstance(document_tokenized, list):
+            return None
+        document_vectorized = self._vectorizer.vectorize(document_tokenized)
```

Comment:
do not create the variable. Return as is

Reply:
done